api/preprints/serializers.py

```diff
@@ -177,6 +184,11 @@ class PreprintSerializer(TaxonomizableSerializerMixin, MetricsSerializerMixin, J
         },
     )
 
+    has_prereg_links = ser.NullBooleanField(required=False)
+    why_no_prereg = ser.CharField(required=False, allow_blank=True, allow_null=True)
+    prereg_links = ser.ListField(required=False)
```

Comment:
This should have a child of URL Field, yes?

Reply:
yes I just accidentally reverted it, sorry.